test/api_getProductRaw_test.dart

```diff
@@ -80,7 +80,7 @@ void main() {
           result.product!.nutrientLevels!.levels[NutrientLevels.NUTRIENT_SALT],
           Level.MODERATE);
       expect(result.product!.countries,
-          'Belgique,France,Allemagne,Pays-Bas,Espagne,Suisse');
+          'Belgique,France,Allemagne,Pays-Bas,Espagne,Suisse, en:world');
```

Comment:
Doesn't look like it's related to my changes, but the test failed without `en:world`.

Reply:
It's because an unrelated change happened on the dev database. I think we should remove that test.